### DISCUSSION TITLE
All Day events  - until text

### DISCUSSION THREAD

**Discussion by mdaffy** [totaldebug/atomic-calendar-revive#429] [Support] [answered]
First - thanks a lot for your cool component.

I have troubles to get the "unitlText" for all day events, which last over several days.
In your example: https://github.com/marksie1988/atomic-calendar-revive/blob/master/.github/img/events-mode.png the last event lasts 2 days and the text "All day, until ... " is shown.

In my card untilText is set like this "untilText: bis"   (bis is German for until)
But nothing (not "bis" and also no date) is show.

Is there any specific option for that?

**Comment by marksie1988** [comment, marked as answer]
Tested and working as expected on latest version.
